src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const totalRentalCost = days * RENT_COST;
+
+  if (days >= LONG_TERM) {
+    return totalRentalCost - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return totalRentalCost - SHORT_TERM_DISCOUNT;
+  }
+
+  return totalRentalCost;
```

Comment:
Returning the `totalRentalCost` at the end is correct as it handles the case when no discounts are applicable.